From b3e5d15327116454a5ff8a74b91cf9eced12bfaf Mon Sep 17 00:00:00 2001
From: Igal Zeltser <izeltser@habana.ai>
Date: Wed, 20 Jan 2021 23:07:17 +0200
Subject: [PATCH] [SW-0] preboot: fixing verification error

adding newline to the macro in order to fix this checkpatch error

ERROR: Macros with complex values should be enclosed in parentheses
24: FILE: preboot/i2cslave_indirect.c:18:
+#define SUPPORTED_OPCODES_MASK    (BIT(HL_I2CI_REQ_OP_DEVICE_INFO) | \
+                BIT(HL_I2CI_REQ_OP_DEVICE_STATUS) | \
                 BIT(HL_I2CI_REQ_OP_CONFIG_ACCESS) | \
                 BIT(HL_I2CI_REQ_OP_FW_LOADER) | \
                 BIT(HL_I2CI_REQ_OP_MEMCPY_CTRL) | \

Change-Id: Ie23f6a7a27dd8ba7a6ab30ea80082c7569d92b94
Signed-off-by: Igal Zeltser <izeltser@habana.ai>
---

diff --git a/preboot/i2cslave_indirect.c b/preboot/i2cslave_indirect.c
index 4c4ede1..eda068f 100644
--- a/preboot/i2cslave_indirect.c
+++ b/preboot/i2cslave_indirect.c
@@ -15,11 +15,13 @@
 #include <hl_i2c_slave_events.h>
 #include "fit.h"
 
-#define SUPPORTED_OPCODES_MASK	(BIT(HL_I2CI_REQ_OP_DEVICE_STATUS) | \
+#define SUPPORTED_OPCODES_MASK	( \
+				BIT(HL_I2CI_REQ_OP_DEVICE_STATUS) | \
 				BIT(HL_I2CI_REQ_OP_CONFIG_ACCESS) | \
 				BIT(HL_I2CI_REQ_OP_FW_LOADER) | \
 				BIT(HL_I2CI_REQ_OP_MEMCPY_CTRL) | \
-				BIT(HL_I2CI_REQ_OP_MEMCPY_DATA))
+				BIT(HL_I2CI_REQ_OP_MEMCPY_DATA) \
+				)
 
 struct i2csi_wdata_chunk_context {
 	u16 offset; /* Chunk offset within the request */
